src/test/PoolDonateTest.sol

```diff
@@ -48,18 +48,18 @@ contract PoolDonateTest is PoolTestBase {
         (,, uint256 reserveBefore1, int256 deltaBefore1) =
             _fetchBalances(data.key.currency1, data.sender, address(this));
 
-        assertEq(deltaBefore0, 0);
-        assertEq(deltaBefore1, 0);
+        require(deltaBefore0 == 0);
```

Comment:
Now that we cannot use asserts, it would be great to have revert reasons in these errors in my opinion. Nothing fancy just something like
`require(deltaBefore0 == 0, 'deltaBefore0 not 0');

or something like that?

Reply:
or `deltaBefore0 not equal to 0`

Reply:
sorry i know that will be a bit boring to do on all these lines 😆 itll just be very helpful for debugging tests

Reply:
I will do it. While I am at it, would it be a good idea to make the other tests in src foundry independent as well?

Reply:
> or `deltaBefore0 not equal to 0`
I have gone with this template for the error messages.